### ISSUE TITLE
Submit OTP automatically after 6th digit

### ISSUE THREAD

**Issue by eliobischof** [zitadel/zitadel#4266]

**Comment by fforootd** [comment]
I think this is a UX topic and should be discussed with @juergrinaldi first

**Comment by juergrinaldi** [comment]
if I understand it correctly from the brief title description, I'm totally down with this, as i also suggested that a couple years ago.

**Comment by juergrinaldi** [comment]
hey, all the cool kids to these nowadays @fforootd you (and me) are just old. :-P

**Comment by Deepizel** [comment]
I could fix this if its still open

**Comment by fforootd** [comment]
> I could fix this if its still open

Yeah this issue is still active, let me know if you need any pointers 😁 

Thank you already

**Comment by hifabienne** [comment]
hei @Deepizel are you still working on this?

**Comment by elinashoko** [comment]
Thanks for filing this issue. As part of an ongoing backlog cleanup, we're closing issues that have had no activity in over 12 months.

If this is still relevant, let us know in a comment (or reopen the issue) and we'll take another look.

This issue will close in 30 days if there's no further activity.
